### ISSUE TITLE
Plannings aren't parsed correctly

### ISSUE THREAD

**Issue by AlexD97** [tecosaur/Org.jl#14]
For example DEADLINE is parsed but DEADLINE: isn't. It should be the other way around.

**Comment by tecosaur** [comment]
I cannot reproduce this.

```julia
julia> org"""
       * some heading
       DEADLINE: [2022-04-19]
       """ |> parsetree
Org Parse Tree
    Heading (*) some heading
        Section
            Planning DEADLINE

julia> org"""
       * some heading
       DEADLINE [2022-04-19]
       """ |> parsetree
Org Parse Tree
    Heading (*) some heading
        Section
            Paragraph
                TextPlain "DEADLINE "
                TimestampInactive
```

**Comment by AlexD97** [comment]
Ah, I see. The problem is that when iterating with `OrgIterator` I do not get the Timestamp of the Planning. How do I get it?

**Comment by tecosaur** [comment]
Fixed in 8e32228cd5842057a30df8d1d621c0c92a9b1fde, which is about to be part of v0.3.1.

**Comment by tecosaur** [comment]
New result, for reference:
```julia

julia> org"""
       * some heading
       DEADLINE: [2022-04-19]
       """ |> parsetree
Org Parse Tree
    Heading (*) some heading
        Section
            Planning DEADLINE
                TimestampInactive
```
